### ISSUE TITLE
Critical security issue caused by proxy-agent dependency

### ISSUE THREAD

**Issue by cptRaclette** [carloscuesta/gitmoji-cli#1193]
### Describe the bug

Hello, 
the proxy-agents package is currently included as a dependency in an outdated version: 5.0.0.
https://github.com/carloscuesta/gitmoji-cli/blob/7d52afc4b28df9cae4ced1937bcaa1fe09c2e1c0/package.json#L57C1-L57C28

This version has a dependency to vm2 which has a [critical security issue](https://github.com/advisories/GHSA-cchq-frgv-rjh5) 

Please upgrade the package.json to reference this package in a newer version (e.g. 6.3.1).

Temporary workaround for packages referencing gitmoji-cli as a dev-dependency AND NOT USING PROXY is adding this to your package.json:
```
  "overrides": {
    "gitmoji-cli": {
      "proxy-agent": "^6.3.1"
    }
  }
  ```

### Reproduction

Activate github advanced security.

### System Info

```Shell
-
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji-cli/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji-cli/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by carloscuesta** [comment]
Hey! @cptRaclette 

Thanks for opening the issue, I'll take a look whenever I have some free time 😊
We have Dependabot security updates enabled on the repo.

Thanks

**Comment by carloscuesta** [comment]
I'll create a release after work! 🙏🏼

**Comment by carloscuesta** [comment]
Released @ `v8.5.0`! Thanks for the help 🥳 

https://github.com/carloscuesta/gitmoji-cli/releases/tag/v8.5.0
